<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>College Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Cambiamos el color de la ventana*/
	QWidget{
		background-color: #656792;
	}

	/*Estilos para el botón*/
	QPushButton{
		background-color: #757DED;
		border-radius: 4px;
		border: 3px solid #101010;
		color: #fff;
		font-family: 'Roboto';
		font-size: 17px;
	}
	
	/*Definimos el estilo para un efecto hover sobre el botón,
	este cambiará su background cuando pasemos el mouse por
	encima*/
	QPushButton:hover{
	background-color:#53F0F3 ;
	}

	/*Definimos los estilos para los QLineEdit*/
	QLineEdit{
		border-radius: 3px;
		border: 2px solid #101010;
		font-color: #020202;
	}

	/*Definimos los estilos para los QLabel*/
	QLabel{
		font-family: 'Monotype Corsiva';
		color: #fff;
	}

	/*Definimos los estilos para los QLabels cuyos nombres son
	'label_usuario' y 'label-password'*/
	#label_usuario, #label_password{
		font-size: 17px;
		color: #212121;
	}
	
	/*Estilo para el QLable cuyo nombre es #label_login*/
	#label_login{
		font-size:30px;
		color: #fff;
	}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="body">
      <item>
       <layout class="QVBoxLayout" name="header">
        <item>
         <widget class="QLabel" name="title_2">
          <property name="font">
           <font>
            <family>Monotype Corsiva</family>
            <pointsize>36</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">text-align: center</string>
          </property>
          <property name="text">
           <string>College Manager</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="role_2">
          <property name="text">
           <string>Administrador</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="userData">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>N° ID: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="nID">
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="name">
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="nav">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="userManagerBtn">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>Administrar Usuarios</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="scheduleBtn">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>Horarios</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="views">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="userManager">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="tableUsers">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="updateUserBtn">
              <property name="text">
               <string>Actualizar Usuario</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addUserBtn">
              <property name="text">
               <string>Agregar Usuario</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteUserBtn">
              <property name="text">
               <string>Borrar Usuario</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="schedule">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="scheduleTable">
            <row>
             <property name="text">
              <string>7:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>13:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>14:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>15:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>16:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>17:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>18:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>19:00</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>20:00</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Lunes</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Martes</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Miercoles</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Jueves</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Viernes</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Sabado</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Domingo</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="adduserPage">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>231</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="form">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Nombre de usuario: </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="usernameLE"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="passwordL">
                <property name="text">
                 <string>Contraseña</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="passwordLE"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="RoleL">
                <property name="text">
                 <string>Rol: </string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="roleCB">
                <item>
                 <property name="text">
                  <string>Estudiante</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Profesor</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Administrador</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="fisrtSuernameL">
                <property name="text">
                 <string>ApellidoPaterno</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="firstSurnameLE"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="SecondSurnameL">
                <property name="text">
                 <string>Apellido Materno:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="secondSurnameLE"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="emailL">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="emailLE"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="phoneL">
                <property name="text">
                 <string>Telefono</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLineEdit" name="phoneLE"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="addressL">
                <property name="text">
                 <string>Direccion</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QTextEdit" name="addressTE"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="nameL">
                <property name="text">
                 <string>Nombre:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="nameLE"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="acceptAddUser">
              <property name="text">
               <string>Crear Usuario</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguracion">
    <property name="title">
     <string>Configuracion</string>
    </property>
    <addaction name="dbConfigAction"/>
   </widget>
   <addaction name="menuConfiguracion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="dbConfigAction">
   <property name="text">
    <string>Base de datos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
